docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -786,8 +786,9 @@ quarkus.oidc-client.credentials.jwt.key-file=privateKey.pem
 ----
 quarkus.oidc-client.auth-server-url=http://localhost:8180/auth/realms/quarkus/
 quarkus.oidc-client.client-id=quarkus-app
-quarkus.oidc-client.credentials.jwt.key-store-file=keystore.jks
-quarkus.oidc-client.credentials.jwt.key-store-password=mypassword
+quarkus.oidc-client.credentials.jwt.tls-configuration-name=oidc-client
```

Comment:
@michalvavrik This is actually wrong, I should've clarified, sorry.

It is not about TLS at all, but about creating a signed JWT token from the keystore, so the key store configuration related to the `credentials.jwt` group must not be changed, please update, sorry again for not making it clear, I kind of referred to it in the issue, but obviously not clearly enough.

Reply:
There is no need to clarify, I was aware and included this because I disagree. Let me remove this.

Reply:
Aware TLS != JWT credentials